specs-go/config.go

```diff
@@ -30,7 +30,12 @@ type ContainerEdits struct {
 type DeviceNode struct {
 	HostPath      string   `json:"hostPath"`
 	ContainerPath string   `json:"containerPath"`
+	Type          string   `json:"type"`
```

Comment:
Should we add `omitempty` to the optional fields?

Reply:
done